src/prax/parser.cr

```diff
@@ -5,7 +5,7 @@ module Prax
   class Parser
     REQUEST_LINE_RE = /\A([A-Z]+)[ \t]+(.+?)[ \t]+(HTTP\/[\d.]+)\Z/
     STATUS_LINE_RE = /\A(HTTP\/[\d.]+)[ \t]+(\d+)[ \t]+(.+?)\Z/
-    HEADER_RE = /([^:]+):\s+(.+)/
+    HEADER_RE = /([^:]+):(\s+(.+))?/
```

Comment:
Yeah, although in the code below, we retrieve the third group, which doesn't include the whitespace: https://regex101.com/r/Nw2Iai/1 I guess we could make the second group non-matching.